docs/en/guides/best-practices/sparse-primary-indexes.md

```diff
@@ -141,7 +141,7 @@ To make this (way) more efficient and (much) faster, we need to use a table with
 
 ### An index design for massive data scales
 
-In traditional relational database management systems, the primary index would contain one entry per table row. For our data set this would result in the primary index - often a <a href="https://en.wikipedia.org/wiki/B%2B_tree" target="_blank">B(+)-Tree</a> data structure - containing 8.87 million entries. Such an index allows the fast location of specific rows, resulting in high efficiency for lookup queries and point updates. Searching an entry in a B(+)-Tree data structure has average time complexity of `O(log2 n)`. For a table of 8.87 million rows, this means 23 steps are required to locate any index entry. This capability comes at a cost: additional disk and memory overheads and higher insertion costs when adding new rows to the table and entries to the index (and also sometimes rebalancing of the B-Tree).
+In traditional relational database management systems, the primary index would contain one entry per table row. This would result in the primary index containing 8.87 million entries for our data set. Such an index allows the fast location of specific rows, resulting in high efficiency for lookup queries and point updates. Searching an entry in a `B(+)-Tree` data structure has an average time complexity of `O(log2 n)`; more precisely, `log_b n = log2 n / log2 b` where `b` is the branching factor of the `B(+)-Tree`. Because `b` is typically between several hundred and several thousand, `B(+)-Trees` are very shallow structures, and few disk-seeks are required to locate records. With 8.87 million rows and a branching factor of 1000, three disk-seeks are needed. This capability comes at a cost: additional disk and memory overheads, higher insertion costs when adding new rows to the table and entries to the index, and sometimes rebalancing of the B-Tree).
```

Comment:
Let's replace `O(log2 n)` by `O(log N)`

Reply:
where `b` is the branching factor of the `B(+)-Tree` and `n` is the number of indexed rows.

Reply:
Could you change `log2` to `log_2`? I also believe that markdown has a LaTeX mode which would display this as a true formula (i.e. print `2` as subscript).